ledger/ledgercore/statedelta.go

```diff
@@ -185,6 +185,37 @@ func (ad NewAccountDeltas) GetData(addr basics.Address) (AccountData, bool) {
 	return ad.accts[idx].AccountData, true
 }
 
+// GetResource looks up a pair of app or asset resources, given its index and type.
+func (ad NewAccountDeltas) GetResource(addr basics.Address, aidx basics.CreatableIndex, ctype basics.CreatableType) (ret AccountResource, ok bool) {
```

Comment:
hmm.. it conflict with `GetResource` that I already implemented in this file.

Reply:
( I don't mind using you implementation, but we should have one function )

Reply:
OK, I am going to delete yours in favor of mine because mine has some extra checks that leave nil pointers on AccountResource when the asset/app holding/param is not found.